features.json

```diff
@@ -64,15 +114,20 @@
 		"relaxedSimd": {
 			"description": "Relaxed SIMD",
 			"url": "https://github.com/WebAssembly/relaxed-simd/tree/main/proposals/relaxed-simd",
-			"phase": 4
+			"phase": 5
 		},
 		"saturatedFloatToInt": {
 			"description": "Non-trapping float-to-int conversions",
 			"url": "https://github.com/WebAssembly/spec/blob/master/proposals/nontrapping-float-to-int-conversion/Overview.md",
 			"phase": 5
 		},
+		"sharedEverythingThreads": {
```

Comment:
This is another one where the spec is in such an early state, and the implementation in V8 is very early and just reflects a small part of what's in the spec.  I don't think I would want to claim in this table (which has no room for nuance) that the spec is "implemented"